solutions/src/main/java/ru/sberbank/school/task08/JacksonSerializableManager.java

```diff
@@ -0,0 +1,68 @@
+package ru.sberbank.school.task08;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.NonNull;
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import ru.sberbank.school.task08.state.MapState;
+import ru.sberbank.school.util.Solution;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+@Solution(8)
+public class JacksonSerializableManager extends SaveGameManager<MapState<GameObject>, GameObject> {
+
+    private ObjectMapper mapper;
+
+    /**
+     * Конструктор не меняйте.
```

Comment:
At-clause should have a non-empty description.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a170314704>*